From 174fe590fd79213d049a5f40bfc4678bdb8a48fc Mon Sep 17 00:00:00 2001
From: Arindam Nath <arindam.nath@amd.com>
Date: Tue, 21 Jul 2015 22:33:53 +0530
Subject: [PATCH 2/2] remove extra forward slash character at include directive

This should avoid build failure in OE.

Upstream Status: None (OE specific)

Signed-off-by: Arindam Nath <arindam.nath@amd.com>
---
 include/llvm/CodeGen/CommandFlags.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/llvm/CodeGen/CommandFlags.h b/include/llvm/CodeGen/CommandFlags.h
index 3c3f770..ddcc1b6 100644
--- a/include/llvm/CodeGen/CommandFlags.h
+++ b/include/llvm/CodeGen/CommandFlags.h
@@ -19,7 +19,7 @@
 #include "llvm/ADT/StringExtras.h"
 #include "llvm/IR/Module.h"
 #include "llvm/MC/MCTargetOptionsCommandFlags.h"
-#include "llvm//MC/SubtargetFeature.h"
+#include "llvm/MC/SubtargetFeature.h"
 #include "llvm/Support/CodeGen.h"
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Host.h"
-- 
1.9.1

